src/components/Alerts/StyledAlert.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { ReactNode } from "react"
+import styled, { css, keyframes } from "styled-components"
+import { AlertContainerProps, AlertContentWrapperProps } from "./AlertInterface"
+import {
+  Magenta80,
+  Yellow80,
+  Green80,
+  Crimson80,
+  Neutral10,
+  Neutral0,
+} from "../../common/styles/palette"
+import { typography } from "../../common/styles/variables"
+import Icon from "../Icon/index"
+
+const isPrefixClassName = (prefixClassName: string | undefined) =>
+  prefixClassName !== "" && prefixClassName !== undefined
+
+const width = {
+  "x-small": 280,
+  small: 350,
+  medium: 490,
+  large: 770,
+}
+
+const background = {
+  standard: Magenta80.hex,
+  warning: Yellow80.hex,
+  success: Green80.hex,
+  error: Crimson80.hex,
+}
+
+const { fontSize: textFontSize, lineHeight: textLineHeight } = typography[14]
+const {
+  fontSize: headingFontSize,
+  lineHeight: headingLineHeight,
+} = typography[18]
+
+export const AlertContainer = styled("div").attrs(
+  (props: AlertContainerProps) => ({
+    className:
+      isPrefixClassName(props.prefixClassName) &&
+      props.prefixClassName + "-knit-alert",
+  })
+)<AlertContainerProps>`
+  width: ${({ size }) => width[size]}px;
+  background: ${({ type }) => background[type]};
+  border-radius: 4px;
```

Comment:
Throughout the PR, px is being used which shouldn't be the case. Please take a look and change all the `rem`